core/app/c/[communitySlug]/ContentLayout.tsx

```diff
@@ -1,10 +1,21 @@
 import type { ReactNode } from "react";
 
-const Heading = ({ title, action }: { title: ReactNode; action: ReactNode }) => {
+const Heading = ({
+	title,
+	action,
+	editFormTitleButton,
```

Comment:
I don't think this prop is necessary on either component. Can you just pass the button in as part of the `title` prop?